.changeset/light-moose-double.md

```diff
@@ -0,0 +1,5 @@
+---
+'@ultraviolet/ui': minor
```

Comment:
It's a breaking theoretically 😐

Reply:
@chambo-e why is it breaking ? I didn't change anything on this package except to make Icon deprecated

Reply:
Ah yeah nvm, I thought you removed the old Icon :)